debezium-connect-rest-extension/pom.xml

```diff
@@ -119,7 +123,7 @@
     <profile>
       <id>assembly</id>
       <activation>
-        <activeByDefault>false</activeByDefault>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Is this necessary and intended?

Reply:
Yes it is! The REST extension now shares code from `debezium-core` and this requires installing it with its dependencies in a "plugin-dir".

Reply:
It's just for the rest-extension module, right? So it won't activate the assembly profile for the entire build (Provided I'm building the entire project)?

Reply:
Yes, this only activates the assembly profile for the `debezium-connect-rest-extension` build.